airgun/views/task.py

```diff
@@ -14,18 +21,51 @@ class TaskReadOnlyEntry(ReadOnlyEntry):
 
 class TasksView(BaseLoggedInView, SearchableViewMixin):
     title = Text("//h1[text()='Tasks']")
+    focus = ActionsDropdown(
+        "//div[./button[@id='tasks-dashboard-time-period-dropdown']]"
```

Comment:
I think we should have just simplified to `//button[@id='tasks-dashboard-time-period-dropdown']`

Reply:
Oh no, current one means "div that contains button with id", yours means "button with id". They are not the same.
We could use this instead: `//button[@id='tasks-dashboard-time-period-dropdown']/parent::div`. I thought about it and decided it's not improvement, so I did not leave any comment.